plutus-benchmark/script-contexts/test/9.6/checkScriptContextEqualityOverhead-20.budget.golden

```diff
@@ -1,2 +1,2 @@
-({cpu: 34304100
-| mem: 214500})
\ No newline at end of file
+({cpu: 37152100
+| mem: 232300})
```

Comment:
@zliu41 is this large enough difference to keep the bug in the dead code eliminator that causes necessary constructors to get eliminated?

Reply:
10% is bad but only a small number of golden files are affected - I'm fine either way. What's your preference?

Reply:
I guess let's keep the bug in then, I think it's quite low priority compared to other things and I don't feel like studying the DCE algo at the moment (or ever haha). I'll reflect this problem in an issue.

Reply:
Sounds good.